Expensify/App
# Design update on FAB and CreateMenu components

@shawnborton @marcaaron 

### Details
Increased the size of the floating action button container, and its offset from the bottom of the screen. Because of that, also increased the offset of the `CreateMenu Popover Modal` (large screens), to make sure it does not go over the button.

Reworked the `Modal` vertical padding calculation, so it fixed the `CreateMenu` problem and possibly more on the future. Now it works like this:
- In `src/styles/getModalStyles`, there are two props `shouldSumTopSafeAreaPadding` and `shouldSumBottomSafeAreaPadding`, which defaults to `false`;
- If the user passes a `paddingTop` or `paddingBottom` to the `modalContainerStyle`, and set the props mentioned to `true`, it will sum the device's `safeAreaPadding` with the specified padding;

This calculation can be seen in the `src/components/Modal.js` file.

### Fixed Issues
Fixes [#1296](https://github.com/Expensify/Expensify.cash/issues/1296)

### Tests
1. Verify that the floating action button size and position looks good,
2. Verify the `CreateMenu Popover Modal` position looks good,
3. Test the `CreateMenu Bottom_docked Modal` in different devices, check if looks good on all. 

### Tested On


### Screenshots

#### Web
![web](https://i.ibb.co/Zm6FnrP/web.png)

#### Mobile Web
![mobile web](https://i.ibb.co/9yHyjw9/mobile-web.png)

#### Desktop
![desktop](https://i.ibb.co/mXVsSy5/desktop.png)

#### iOS
![ios](https://i.ibb.co/4f6xGLS/ios.png)

#### Android
![android](https://i.ibb.co/sgj03Qh/android.png)
